src/mscorlib/corefx/System/IO/PathInternal.Unix.cs

```diff
@@ -43,22 +43,6 @@ internal static bool IsDirectorySeparator(char c)
         }
 
         /// <summary>
-        /// Returns true if the path is too long
-        /// </summary>
-        internal static bool IsPathTooLong(string fullPath)
-        {
-            return fullPath.Length >= Interop.Sys.MaxPath;
-        }
-
-        /// <summary>
-        /// Returns true if the directory is too long
-        /// </summary>
-        internal static bool IsDirectoryTooLong(string fullPath)
-        {
-            return fullPath.Length >= Interop.Sys.MaxPath;
-        }
```

Comment:
@JeremyKuhne I interpret that to mean there is a good reason for both, and I can continue to delete what's not used by coreclr.